app/components/fandom-app-smart-banner.js

```diff
@@ -37,6 +38,18 @@ export default Component.extend({
 			: this.get('i18n').t('fandom-app-banner.google-play');
 	}),
 
+	/**
+	 * Obtains domain we should cookies on. Since Fandom App SmartBanner is global for all the wikis,
+	 * we should set it to all subdomains of wikia.com, wikia-dev.us or wikia.dev-pl
+	 *
+	 * @returns {string}
+	 */
+	cookieDomain: computed(() => {
+		const domainSegmentsReversed = window.location.hostname.split('.').reverse();
+
+		return `.${domainSegmentsReversed[1]}.${domainSegmentsReversed[0]}`;
```

Comment:
You can use `runtimeConfig.cookieDomain` which returns the same thing.

Reply:
+1